keras/applications/convnext.py

```diff
@@ -0,0 +1,647 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+# pylint: disable=invalid-name
+# pylint: disable=missing-docstring
+# pylint: disable=g-classes-have-attributes
+# pylint: disable=g-direct-tensorflow-import
+"""ConvNeXt models for Keras.
+
+References:
+
+- [A ConvNet for the 2020s](https://arxiv.org/abs/2201.03545)
+  (CVPR 2022)
+"""
+
+from keras import backend
+from keras import layers
+from keras import utils
+from keras import Sequential
+from keras.applications import imagenet_utils
+from keras.engine import training
+
+import tensorflow.compat.v2 as tf
+from tensorflow.python.util.tf_export import keras_export
+
+BASE_WEIGHTS_PATH = "https://storage.googleapis.com/convnext-tf/keras-applications-temp/convnext/"
```

Comment:
you can now change this to:

BASE_WEIGHTS_PATH = "https://storage.googleapis.com/tensorflowkeras-applications/convnext/"